code/ship/ship.cpp

```diff
@@ -10334,6 +10334,8 @@ int ship_fire_primary(object * obj, int stream_weapons, int force)
 				if (winfo_p->wi_flags3 & WIF3_APPLY_RECOIL){
 					firepoint_list = new vec3d[numtimes * points];
 					vm_vec_zero(&total_impulse);
+				} else {
+					firepoint_list = nullptr;
```

Comment:
Maybe C++11 support needs to be enabled with a compiler switch just like on linux?
[This SO answer](http://stackoverflow.com/a/7791906) says that it needs to be enabled in the project properties.